config/application.rb

```diff
@@ -98,7 +98,7 @@ def self.config
   POPULAR_DAYS_LIMIT = 70.days
   PROTOCOL = config["protocol"]
   REMEMBER_FOR = 2.weeks
-  SEARCH_INDEX_NAME = "rubygems-#{Rails.env}".freeze
+  SEARCH_INDEX_NAME = ENV.fetch("SEARCH_INDEX_NAME", "rubygems-#{Rails.env}".freeze)
```

Comment:
Why not use config also?

Reply:
Good question. I think it depends on if we would change it across servers of the same env.